public/view/overlay-controls/selected-admins-info.jsx

```diff
@@ -4,6 +4,11 @@ var _ = require('lodash');
 require('./selected-admins-info.css');
 
 var SelectedAdminsInfo = React.createClass({
+
+  commify: function( x ) {
+    return parseInt(x).toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+  },
```

Comment:
Also, pretty sad that `toLocaleString` doesn't work :-/ https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString